### ISSUE TITLE
Add ability to plot overviews when using Rasterio node

### ISSUE THREAD

**Issue by rmeyers4** [creare-com/podpac#433]
**Description**
When working with a large GeoTIFF dataset that has overviews, it can be useful to see a thumbnail of the whole dataset to understand the variations in the data. I'm not aware of a way to plot overviews in PODPAC, and creating a set of coordinates using `clinspace` that spans a large dataset can lead to running out of memory. 

**Describe the solution you'd like**
Add a method to the PODPAC Rasterio node that allows overviews to be plotted. This can be done in `rasterio` with something like this, but it could be nice to abstract this to a method.
```
with rasterio.open('s3://soilmap/data/ssurgo/soc0_5.tif') as src:
    oviews = src.overviews(1)
    oview = oviews[2]
    thumbnail = src.read(1, out_shape=(1, int(src.height // oview), int(src.width // oview)))
plt.imshow(thumbnail)
plt.colorbar()
```
I propose adding a property and a method to the Rasterio node that would show available overviews and plot them on request.
`node.overviews` would return a list of available overviews, and `node.plot_overview(<overview>)` would plot the selected overview in `matplotlib`, using any `Styles` associated with the node.

**Comment by mpu-creare** [comment]
YES! This is on my TODO list. For now if you use the `nearest_preview` interpolation setting, overview data will be automatically pulled based one the coarseness of the data request.

**Comment by mpu-creare** [comment]
@jmilloy -- this is bubbled up to be high priority. Can you have a look at this?

Basically, whenever a user asks for a coarser set of coordinates, prefer to use the overviews, if available.  We should add a flag, something like `prefer_overviews=True` 

Rasterio does *some* of this automatically, but only for strided access of data -- so not general at all.

**Comment by mpu-creare** [comment]
This is helpful:
```
ovr = rasterio.open(self.source, overview_level=0)
```
You have to open the file again, with the overview explicitly specified as above. To find what 'level' the overview represent, you also need `rasterio.open(source).overviews(self.band)`

**Comment by jmilloy** [comment]
@mpu-creare What is the status on this? Should I work on this next, or shall we discuss priorities sometimes soon?

**Comment by mpu-creare** [comment]
ah! I did this already. Closed via #475
